pallets/block-rewards/src/mock.rs

```diff
@@ -237,35 +229,38 @@ frame_support::parameter_types! {
 	pub const BlockRewardCurrency: CurrencyId = CurrencyId::Staking(BlockRewardsCurrency);
 	pub const StakeAmount: Balance = cfg_types::consts::rewards::DEFAULT_COLLATOR_STAKE;
 	pub const CollatorGroupId: u32 = cfg_types::ids::COLLATOR_GROUP_ID;
+	pub const TreasuryPalletId: PalletId = cfg_types::ids::TREASURY_PALLET_ID;
 }
 
 impl pallet_block_rewards::Config for Test {
 	type AdminOrigin = EnsureRoot<AccountId>;
 	type AuthorityId = UintAuthorityId;
 	type Balance = Balance;
-	type Beneficiary = RewardRemainderMock;
-	type Currency = Tokens;
 	type CurrencyId = CurrencyId;
 	type ExistentialDeposit = ExistentialDeposit;
 	type MaxChangesPerSession = MaxChangesPerSession;
 	type MaxCollators = MaxCollators;
+	type Rate = Rate;
 	type Rewards = Rewards;
 	type RuntimeEvent = RuntimeEvent;
 	type StakeAmount = StakeAmount;
 	type StakeCurrencyId = BlockRewardCurrency;
 	type StakeGroupId = CollatorGroupId;
+	type Time = MockTime;
+	type Tokens = Tokens;
+	type TreasuryPalletId = TreasuryPalletId;
 	type Weight = u64;
 	type WeightInfo = ();
 }
 
 pub(crate) fn assert_staked(who: &AccountId) {
 	assert_eq!(
 		// NOTE: This is now the ED instead of 0, as we collators need ED now.
-		<Test as Config>::Currency::balance(<Test as Config>::StakeCurrencyId::get(), who),
+		<Test as Config>::Tokens::balance(<Test as Config>::StakeCurrencyId::get(), who),
```

Comment:
Super NIT:
```suggestion
		Tokens::balance(BlockRewardCurrency::get(), who),
```

in some occurrences.